libs/blocks/icon-block/icon-block.css

```diff
@@ -42,6 +42,16 @@
   width: 100%;
 }
 
+.five-up .foreground .text-content {
```

Comment:
This is not specific to .icon-block - so this might affect other parts of the page.  Can you scope it to the icon block?

Reply:
just saw that it's existing css code that was moved, but I'm still curious as to why

Reply:
The code was moved up to resolve StyleLint errors.

Reply:
All of the other `.five-up` selectors are still scoped to the icon block - e.g:
`.five-up .icon-block.fullwidth {`
but this one isn't, and could affect parts of the page unrelated to the icon-block

Reply:
@chrischrischris i have scoped this to icon-block.